packages/dds/tree/api-report/tree.api.md

```diff
@@ -1652,19 +1648,19 @@ export interface TreeApi {
 }
 
 // @public
-export interface TreeArrayNode<T extends ImplicitAllowedTypes = ImplicitAllowedTypes> extends TreeNode, TreeArrayNodeBase<TreeNodeFromImplicitAllowedTypes<T>, InsertableTreeNodeFromImplicitAllowedTypes<T>, TreeArrayNode> {
+export interface TreeArrayNode<TAllowedTypes extends ImplicitAllowedTypes = ImplicitAllowedTypes> extends TreeNode, TreeArrayNodeBase<TreeNodeFromImplicitAllowedTypes<TAllowedTypes>, InsertableTreeNodeFromImplicitAllowedTypes<TAllowedTypes>, TreeArrayNode> {
 }
 
 // @public
 export const TreeArrayNode: {
-    inline: <T>(content: Iterable<T>) => IterableTreeListContent<T>;
+    inline: <T>(content: Iterable<T>) => IterableTreeArrayContent<T>;
```

Comment:
@CraigMacomber - Since you're right here, would you mind also renaming 'inline' to 'spread'?

Here's my rationale from 11/28:

> Rename .inline -> .spread
> The only practical way to exhaust the stack is using the spread operator:
>  
> notes.insertAtStart(...myHugeArray);
>  
> The fix is to replace the spread operator with the spread method, which behaves identically:
>  
> notes.insertAtStart(spread(myHugeArray));

Here is Noah's feedback:

> Yea, good idea. It's actually slightly cooler than JS spread because you can do list.insertAtStart(a, spread(...), b, spread(...) c) and end up with [a, ..., b, ..., c] , but that doesn't mean we need to pick a different name. I think that users who do see the function for the first time will more quickly intuit what it does from the name spread vs. inline. Ship it!

Reply:
Done.